bucket/quicklook.json

```diff
@@ -9,6 +9,9 @@
     ],
     "url": "https://github.com/QL-Win/QuickLook/releases/download/3.7.3/QuickLook-3.7.3.zip",
     "hash": "e1f964fe2490b229359bd6900f801998aef8af25781d8786ceda197206404e0f",
+    "bin": [
+        "QuickLook.exe"
+    ],
```

Comment:
```suggestion
    "bin": "QuickLook.exe",
```

Reply:
Thanks for your suggestion. I have updated it.